epoch/api/chain_ws_api.md

```diff
@@ -21,7 +20,7 @@ The WebSocket API provides the following actions:
   | Name | Type | Description | Required |
   | ---- | ---- | ----------- | -------- |
   | type | string | `block` | Yes |
-  | height | number | Height of the block to fetch | Yes |
```

Comment:
It looks still present in https://github.com/aeternity/epoch/pull/1293 i.e. at https://github.com/aeternity/epoch/blob/9a09d588847ac5c37ab7bf834d8b15e26f45a5aa/apps/aehttp/src/ws_int_dispatch.erl#L34-L35 ?

Reply:
It's still present, but not working reliably as block by height can return key or micro block, so deleted from the documentation until the new HTTP is introduced.